packages/snap-client/src/Client/transforms/searchRequest.ts

```diff
@@ -29,11 +29,11 @@ transformSearchRequest.sorts = (request: SearchRequestModel = {}) => {
 		}
 
 		if (!sort.field || !sort.direction) {
-			throw 'valid sort requires field and direction';
+			return {};
```

Comment:
Looking at this code a bit closer I realize that you may need to return acc instead of the empty object here. Might want to take a closer look at this.